Tools/MiniBrowser/mac/WK2BrowserWindowController.m

```diff
@@ -50,6 +51,23 @@
 static const int testHeaderBannerHeight = 42;
 static const int testFooterBannerHeight = 58;
 
+static CALayer *makePointerLockBanner(void)
```

Comment:
Maybe add this as an instance variable in `@implementation WK2BrowserWindowController`? Nice to avoid singletons when possible.

Reply:
Fixed in [9bcdba2](https://github.com/WebKit/WebKit/pull/19617/commits/9bcdba2b0ce081f28aca988bc21db7e9f711e0d1)